qt_ui/windows/finances/QFinancesMenu.py

```diff
@@ -59,7 +59,6 @@ def add_total(self, game, income, player):
         )
         budget = game.coalition_for(player).budget
         self.add_row(middle="Balance", right=f"<b>{budget:.1f}M</b>")
-        self.setRowStretch(next(self.row), 1)
```

Comment:
With no stretch it'll look very odd when there are few entries, or if someone resizes the window to make it taller. The mistake here is that the stretch always comes after the total, but at some point `total_at_top` was introduced. That's no longer actually used either. Could you instead remove the `total_at_top` parameter, unconditionally put it at the top, and unconditionally add the stretch at the end?

Reply:
Thanks for the info, I've updated the PR to remove the `total_at_top` parameter, hard code the current behavior and move the row stretch to the very bottom. The screenshots in the PR have been updated and the window looks much better.